featuretools/feature_base/feature_base.py

```diff
@@ -370,47 +377,54 @@ class DirectFeature(FeatureBase):
         a feature value from a parent entity"""
     input_types = [Variable]
     return_type = None
+    # Direct features are applied through forward relationship paths.
+    path_is_forward = True
```

Comment:
I added this so that the `_add_feature_to_trie` did not have to check the feature type. But we do use `isinstance` in many places so I can go that route if it is the preferred style.